lib/benchee.ex

```diff
@@ -102,5 +103,10 @@ for {module, moduledoc} <- [{Benchee, elixir_doc}, {:benchee, erlang_doc}] do
     See `Benchee.ScenarioLoader.load/1`
     """
     defdelegate load(suite), to: Benchee.ScenarioLoader
+
+    @doc """
+    See `Benchee.Profile.profile/1`
+    """
```

Comment:
I think that's even fixed in newer versions so we might remove it, but probably in another PR :)

Reply:
it is fixed :) I can remove it later or now just fyi (I think you get the changes from master)